package.json

```diff
@@ -70,8 +70,8 @@
     "cozy-logger": "^1.9.0",
     "cozy-realtime": "4.0.5",
     "cozy-scripts": "6.3.8",
-    "cozy-sharing": "^4.5.10",
-    "cozy-ui": "^70.6.0",
+    "cozy-sharing": "^4.7.0",
+    "cozy-ui": "^77.6.0",
```

Comment:
no BC to deal with?

Reply:
sry, but what do you mean by BC ?

Reply:
breaking changes. https://github.com/cozy/cozy-ui/blob/master/CHANGELOG.md between 70 and 76. Notes is not concerned by them?

Reply:
Notes is not concerned because it doesn't use `List`, `Dropdown`, `BottomSheet`from my research.